integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/ratings/OffendingHistorySpecification.groovy

```diff
@@ -68,7 +68,6 @@ class OffendingHistorySpecification extends GebReportingSpec {
     db.getData(12).status == ["STARTED"]
   }
 
-
```

Comment:
do we have some end to end test coverage on the display outcome? It feels like the end html should be tested